A14-channelz.md

```diff
@@ -728,8 +730,27 @@ message ChannelData {
   google.protobuf.Timestamp last_call_started_timestamp = 7;
 }
 
+message ChannelTraceEvent {
+  string description = 1;
+  string error = 2;
```

Comment:
Should this be a Status?  https://github.com/googleapis/googleapis/blob/master/google/rpc/status.proto#L80

They can express the kinds of errors the application will see, and they have room for additional details.

Reply:
I initially meant this to be `grpc_error`, but seeing as that is language specific, I think that Status is a better choice here